packages/cfd/src/Containers/cfd-password-success-icon.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { CFD_PLATFORMS, PRODUCT } from '../Helpers/cfd-config';
+import TradingPlatformIcon from '../Assets/svgs/trading-platform';
+import { TProducts } from '../Components/props.types';
+
+type TCFDPasswordSuccessIconProps = {
+    platform: string;
+    type?: string;
+    show_eu_related_content: boolean;
+    product?: TProducts;
+};
+
+const CFDPasswordSuccessIcon = ({ platform, type, show_eu_related_content, product }: TCFDPasswordSuccessIconProps) => {
```

Comment:
conditions hell :)
can we some how refactor the code?
maybe we could define the icon string separately and then just return something like
<TradingPlatformIcon icon={defined_icon_string} /> and same for <Icon />